src/Model.php

```diff
@@ -150,15 +150,16 @@ class Model implements \IteratorAggregate
 
     /**
      * After loading an active record from DataSet it will be stored in
-     * $data property and you can access it using get(). If you use
+     * $data property, and you can access it using get(). If you use
      * set() to change any of the data, the original value will be copied
      * here.
      *
      * If the value you set equal to the original value, then the key
      * in this array will be removed.
      *
-     * The $dirty data will be reset after you save() the data but it is
-     * still available to all before/after save handlers.
+     * Behaviour in Model::save():
```

Comment:
private variable does not need any docs, as the class code is enought together with the tests to document it

Reply:
I am ok with both - either have a correct comment or no comment at all. Do as you like, I would prefer the correct comment as it describes the behaviour and so new users need to read less code.

Reply:
The problem with too much phpdoc comments is they get easily outdated and only increase LoC.

If you want this to be documented, please update docs/. Please make sure the comment is crisply clear as the current proposal is not much helpful IMHO, as the described behaviour is expected and probably even already documented in docs/.